qubesu2f/tools/__init__.py

```diff
@@ -29,7 +29,10 @@
 import pathlib
 import sys
 
-import u2flib_host.u2f  # pylint: disable=import-error
+try:
+    import fido2  # pylint: disable=import-error
```

Comment:
I don't think just this would be enough, below are still references to `u2flib_host.u2f`. Similar in the tests file.

Reply:
At the very least here: https://github.com/QubesOS/qubes-app-u2f/pull/30/files#diff-073a291566f3ad7b334c58c32c67a04fdaa27d49d746b2a4ddd8aa340359b63bR62